components/anchor/Anchor.tsx

```diff
@@ -51,9 +51,9 @@ export interface AnchorProps {
   children?: React.ReactNode;
   offsetTop?: number;
   bounds?: number;
-  affix?: boolean;
-  showInkInFixed?: boolean;
-  getContainer?: () => AnchorContainer;
+  affix: boolean;
+  showInkInFixed: boolean;
+  getContainer: () => AnchorContainer;
```

Comment:
Don't remove `?` that make props required.

Reply:
If you've defined the `defaultProps` in the class, typescript is clever enough now to mark them as optional when the component is used. 
<img width="801" alt="image" src="https://user-images.githubusercontent.com/10626756/80169488-eb236280-8628-11ea-90c4-4514e20b473a.png">

Reply:
We can remove `defaultProps` of Anchor. `?` is correct and should not be removed.

Reply:
@zombieJ Do you mean that the exported type need to preserve the optional parameter for consumers?